@here/harp-mapview/lib/MapView.ts

```diff
@@ -1877,28 +1877,23 @@ export class MapView extends THREE.EventDispatcher {
         const limitedTilt = Math.min(89, tiltDeg);
         // MapViewUtils#setRotation uses pitch, not tilt, which is different in sphere projection.
         // But in sphere, in the tangent space of the target of the camera, pitch = tilt. So, put
-        // the camera on the target, so the tilt can be passed to setRotation as a pitch.
-        this.geoCenter = target;
-        MapViewUtils.setRotation(this, -azimuthDeg, limitedTilt);
-        // Then deduce the actual geoCenter.
-        this.geoCenter = MapViewUtils.getCameraCoordinatesFromTargetCoordinates(
+        // the camera on the target, so the tilt can be passed to getRotation as a pitch.
```

Comment:
The 3 lines of comments are now wrong, they can be removed.

Reply:
The 3 lines can be removed. It was to justify was the geoCenter was set to the target to then call `setRotation`, but this is not done anymore.

Reply:
Yes, and now they are there to justify why the target is actually a parameter to this method. Maybe I reword it a bit more.